internal/syncasync/sync_async_bridge.go

```diff
@@ -44,6 +44,8 @@ type Bridge interface {
 	WaitForReply(ctx context.Context, ns string, id *fftypes.UUID, send RequestSender) (*fftypes.MessageInOut, error)
 	// WaitForMessage waits for a message with the supplied ID
 	WaitForMessage(ctx context.Context, ns string, id *fftypes.UUID, send RequestSender) (*fftypes.Message, error)
+	// WaitForIdentity waits for an identity with the supplied ID
+	WaitForIdentity(ctx context.Context, ns string, id *fftypes.UUID, send RequestSender) (*fftypes.Identity, error)
```

Comment:
Realized I needed to add full sync-async support for `IdentityClaim` because either message could be confirmed first.